TAF/utils/scripts/docker/get-compose-file.sh

```diff
@@ -32,6 +32,8 @@ if [ "$USE_RELEASE" = "pre-release" ]; then
   if [ "$USE_SECURITY" = "-security-" ]; then
     sed -i "/ROUTES_RULES-ENGINE_HOST/a \ \ \ \ \ \ ADD_PROXY_ROUTE: 'modbusdevice.http://edgex-device-modbus:49991'" \
     docker-compose.yaml
+    sed -i 's/\USERNAME_PLACEH_OLDER/${MQTT_USER}/g' docker-compose.yaml
+    sed -i 's/\PASSWORD_PLACE_HOLDER/${MQTT_PWD}/g' docker-compose.yaml
```

Comment:
These are not needed in secure mode since those place holders are for InsecureSecrets. These would be needed in the non-secure path if securing the broker in non-secure mode.. Also looks like we have typo to fix in `compose builder` source files... ;-)

```
      WRITABLE_INSECURESECRETS_MQTT_SECRETS_PASSWORD: PASSWORD_PLACE_HOLDER
      WRITABLE_INSECURESECRETS_MQTT_SECRETS_USERNAME: USERNAME_PLACEH_OLDER
```
If not needed in non-secure mode, we can remove the placeholder overrides.

Reply:
@lenny-intel Removed the override lines of username and password from the `get-compose-file.sh`. And yes, we don't use username and password in non-secure mode, both of them can be removed. Thank you.